README.md

```diff
@@ -32,58 +38,44 @@ ReactDOM.render(
 );
 
 ```
-Once you are logged into Sentry, go [here](https://docs.sentry.io/platforms/javascript/react/) to find your `Sentry.init` statement (prefilled with your Sentry `dsn` value). Once you're logged into FullStory, you can find your org id in the [settings page](https://app.fullstory.com/ui/1ENq/settings/general). It will be the value set for `window['_fs_org']`.
+Once you are logged into Sentry, go [here](https://docs.sentry.io/platforms/javascript/react/) to find your `Sentry.init` statement (prefilled with your Sentry `dsn` value). Copy that value into the `REACT_APP_SENTRY_DSN` field in your `.env` file. Next, find the slug of your organization from the URL of your Sentry account. Example: `https://sentry.io/organizations/fullstory/` where `fullstory` would be the organization slug. Copy that value into `REACT_APP_SENTRY_ORG` in your `.env` file.
+
+Once you're logged into FullStory, you can find your org id in the [settings page](https://app.fullstory.com/ui/1ENq/settings/general). It will be the value set for `window['_fs_org']`. Copy that value into `REACT_APP_FULLSTORY_ORG` of your `.env` file.
 
 ## How FullStory links with Sentry
-FullStory’s [`FS.getCurrentSessionURL`](https://developer.fullstory.com/current-session-url) API function generates a session replay URL for a particular moment in time. These URLs are deep links that can be shared with other tools and services. Session URLs are embedded into Sentry events when [extra context](https://docs.sentry.io/enriching-error-data/context/?platform=javascript#extra-context) is configured by providing a value for `event.extra.fullstory` in the [beforeSend](https://docs.sentry.io/error-reporting/configuration/filtering/?platform=javascript#before-send) hook. The [`src/api/error.js`](https://github.com/patrick-fs/fs-react-redux-sentry/blob/master/src/api/error.js) module puts it all together.
+FullStory’s [`FS.getCurrentSessionURL`](https://developer.fullstory.com/current-session-url) API function generates a session replay URL for a particular moment in time. These URLs are deep links that can be shared with other tools and services. Session URLs are embedded into Sentry events when [extra context](https://docs.sentry.io/enriching-error-data/context/?platform=javascript#extra-context) is configured by providing a value for `event.contexts.fullstory` in the [beforeSend](https://docs.sentry.io/error-reporting/configuration/filtering/?platform=javascript#before-send) hook. The [Sentry-FullStory](https://github.com/getsentry/sentry-fullstory) integration package puts it all together.
 
```

Comment:
How about adding in a mention of FS.event integration?

Original copy for reference:
> We’re also using the FullStory [custom events API](https://help.fullstory.com/develop-js/363565-fs-event-api-sending-custom-event-data-into-fullstory) to send error data into FullStory. This lets us search for all users that experienced errors on the Search Hacker News app.

Reply:
@patrick-fs I removed that because that part is now hidden from the developer since it's contained in the package. But I will happily add it back.

Reply:
That's fine. The Integration actually abstracts _all the things_ so maybe we update that README with integration details and simply have a statement like "if you want to know more about how the integration works, go to sentry-fullstory"